packages/lit-node-client-nodejs/src/lib/helpers/handle-bls-response.ts

```diff
@@ -0,0 +1,26 @@
+import { log } from '@lit-protocol/misc';
+import { BlsResponseData } from '@lit-protocol/types';
+
+/**
+ * Handles the response data from BLS  signature scheme.
+ * @param responseData - The response data from BLS signature scheme.
+ * @returns An array of signed data.
+ * @throws Error if no data is provided.
+ */
+export function handleBlsResponseData(
```

Comment:
The interface `BlsResponseData` is manually defined instead of automatically generated, in this case the unit test would at least have the original structure of the data when `handleBlsResponseData` throws an error